testing/mod.ts

```diff
@@ -165,6 +166,19 @@ function report(result: TestResult): void {
   result.printed = true;
 }
 
+function printFailedSummary(results: TestResults): void {
+  results.cases.forEach(
+    (v): void => {
+      if (!v.ok) {
+        console.error(`${RED_BG_FAIL} ${red(v.name)}`);
+        if (Deno.args.includes("--verbose")) {
```

Comment:
I'm currently writing test runner which will abstract away most of the options for testing framework. Could you remove this conditional for now and always print error?

Reply:
okay, i see! yes ive fixed it now